tests/shared/src/test/scala-2.13/scala/meta/tests/parsers/dotty/Scala3PositionSuite.scala

```diff
@@ -9,35 +9,47 @@ class Scala3PositionSuite extends BasePositionSuite(dialects.Scala3) {
   checkPositions[Type]("A | B")
   checkPositions[Type](
     "[X] =>> (X, X)",
-    """|Type.Bounds [X@@] =>> (X, X)
+    """|Type.ParamClause [X]
+       |Type.ParamClause [X@@] =>> (X, X)
```

Comment:
my guess is that Type.Param can have its own type parameters, for instance `F[_]`, hence there's an empty clause. would you prefer the clause to be non-empty and use Option?

Reply:
I think it's fine in the current case, since we can't have []. Option might be more useful for `method` vs `method()` so Term.Param.

I just thought that the structure here has only one ParamClause, but I see now that it's not the case. Nvm then.

Reply:
so, ok to keep as-is?

(for Term.Param, Option is not needed, but we'll discuss that in the next PR.)

Reply:
Yep, I think it's fine.